react-native-community/react-native-maps
# Implement mapPadding in iOS maps

### Does any other open PR do the same thing?



No

### What issue is this PR fixing?

During my migration to Apple Maps, I noticed that mapPadding isn't supported. This PR fixes that.

### How did you test this PR?



- Create a plain MapView
- Add some children to MapView
- Add mapPadding
- Padding should work just fine, just like on Google Maps
